apps/prairielearn/src/sync/fromDisk/assessments.ts

```diff
@@ -133,20 +130,15 @@ function getParamsForAssessment(
             maxAutoPoints: alternative.maxAutoPoints ?? question.maxAutoPoints ?? null,
             autoPoints: alternative.autoPoints ?? question.autoPoints ?? null,
             manualPoints: alternative.manualPoints ?? question.manualPoints ?? null,
-            forceMaxPoints: _.has(alternative, 'forceMaxPoints')
-              ? alternative.forceMaxPoints
-              : _.has(question, 'forceMaxPoints')
-                ? question.forceMaxPoints
-                : false,
-            triesPerVariant: _.has(alternative, 'triesPerVariant')
-              ? alternative.triesPerVariant
-              : _.has(question, 'triesPerVariant')
-                ? question.triesPerVariant
-                : 1,
-            advanceScorePerc: alternative.advanceScorePerc,
-            gradeRateMinutes: _.has(alternative, 'gradeRateMinutes')
-              ? alternative.gradeRateMinutes
-              : questionGradeRateMinutes,
+            forceMaxPoints: alternative.forceMaxPoints ?? question.forceMaxPoints ?? false,
+            triesPerVariant: alternative.triesPerVariant ?? question.triesPerVariant ?? 1,
+            advanceScorePerc:
+              alternative.advanceScorePerc ??
+              question.advanceScorePerc ??
+              zone.advanceScorePerc ??
+              assessment.advanceScorePerc ??
+              0,
+            gradeRateMinutes: alternative.gradeRateMinutes ?? questionGradeRateMinutes,
             canView: alternative?.canView ?? questionCanView,
```

Comment:
@nwalters512 This line implies alternatives have `canView`/`canSubmit` fields. is that intentional, or a bug?

Reply:
Addressed in 57735c1.